test/e2e/admission_controller/admission_controller_suite_test.go

```diff
@@ -197,9 +196,8 @@ var _ = BeforeSuite(func() {
 	kubeClient = k8s.KubeCtl{}
 	Expect(kubeClient.SetClient()).To(BeNil())
 
-	annotation = "ann-" + common.RandSeq(10)
 	yunikorn.EnsureYuniKornConfigsPresent()
-	yunikorn.UpdateConfigMapWrapper(oldConfigMap, "", annotation)
```

Comment:
Could you please trace the git to find out which commit makes it unused? I guess https://github.com/apache/yunikorn-k8shim/commit/ec9d10d60dc4e8529a3ffa346502d2242bc7d4d0